doc/api/addons.markdown

```diff
@@ -1077,14 +1078,14 @@ Test in JavaScript by running:
 const addon = require('./build/Release/addon');
 ```
 
-[online]: https://v8docs.nodesource.com/
```

Comment:
I like them in order but I don't think other files are doing this? So I'd just leave the order as it is (as reorder all files are probably not feasible).